.github/workflows/jepsen.yml

```diff
@@ -7,8 +7,6 @@ concurrency:
 on: # yamllint disable-line rule:truthy
   schedule:
     - cron: '0 */6 * * *'
-  workflow_dispatch:
-  workflow_call:
```

Comment:
we could let the `call` go, but `dispatch` is useful to have this button
![image](https://github.com/ClickHouse/ClickHouse/assets/3025537/bcf3570d-3ba6-43a5-be9c-4b8e3410d3f9)